frontends/web/Makefile

```diff
@@ -16,6 +16,12 @@ dev:
 	yarn run dev
 lint:
 	yarn run lint
+jstest:
+	yarn run test
+jstest-watch:
+	yarn run test --watch
+jstest-cover:
+	yarn run test --coverage
```

Comment:
one has to `cd frontends/web` to be able to use `make jstest-watch` or can you call from project root?

in this case I would not add it to Makefile but rather to package.json scripts, maybe similar to  https://github.com/thisconnect/bitbox-wallet-app/blob/frontend/tests/frontends/web/package.json#L10

Reply:
Sure, but moving these to package.json would still require to `cd frontends/web`. The reason being that this is a specific task and you won't need it most of the time, unless you're a heavy web developer and spend most of your time in frontends/web anyway.

It just seemed to be more consistent with the existing commands: everything is run via `make <something>` so did that.

Reply:
Don't you like explicit flags to understand what's actually going to happen? :)

Reply:
I think it is fine and don't mind at all. 👍 

> Don't you like explicit flags to understand what's actually going to happen

to understand what will happen when calling `npm test`/`yarn test` there is only 1 thing you can do... read the package.json (rtfp is the new rtfm) :)